### DISCUSSION TITLE
"Error: Billing plan is invalid" while creating database using cli

### DISCUSSION THREAD

**Discussion by sean-ahn** [planetscale/discussion#498] [Bug Reports] [answered]
Hello,

I am currently using the Hobby plan.

I encountered an issue while attempting to create a database using the `pscale` CLI. The error message "Error: Billing plan is invalid." appeared.

I have included the full output from my terminal below:

```
➜  workspace pscale --version
pscale version 0.150.0 (build date: 2023-07-13T23:55:37Z commit: 99f1c48)
➜  workspace pscale org show
sean9509 (from file: /Users/seanahn/.config/planetscale/pscale.yml)
➜  workspace pscale db list
No databases have been created yet.
➜  workspace pscale db create prisma-playground

Error: Billing plan is invalid.
```

**Comment by sean-ahn** [comment, marked as answer]
I successfully created a database using the `--plan` option by executing the following command:
```
pscale database create prisma-playground --plan hobby
```

It appears that the `--plan` option was recently introduced in version 0.149.0, as indicated in this URL: https://github.com/planetscale/cli/releases/tag/v0.149.0.

It would be more convenient if the `--plan` option were automatically filled in by default, rather than having to input it every time.

I've found a related code line in your https://github.com/planetscale/cli repository:
> https://github.com/planetscale/cli/blob/99f1c483300c9b625e343823107a7176b6cd055d/internal/cmd/database/create.go#L105

**Reply by jgreet** [reply]
Thanks for the report! We fixed this on the API side, no pscale upgrade necessary. 

In this situation where we're defaulting to the hobby plan again.
